### ISSUE TITLE
Try to gather people from XOAI forks to gain more momentum

### ISSUE THREAD

**Issue by poikilotherm** [gdcc/xoai#9]
Hi 
- @IQSS: @landreev, (#2)
- @GBIF: @mike-podolskiy90, @MattBlissett, @fmendezh, @cgendreau (#8)
- @LOCKSS: @thibgc (#7)
- @ITWOI

you have been working on patches and bug reports for XOAI in the (not so) recent past. As Dataverse software also uses XOAI to provide OAI-PMH, the community would like to [provide a modernized Java 11+ library](https://github.com/gdcc/xoai/tree/branch-5.0) (#1) to be ready for the future.

Would it make sense to you to join forces? 
It would be great to gather people here first (leave a note) and then use some public space to chat (issues ain't great for that).

If you aren't involved in this business anymore, please just unsubscribe from this issue to turn of notifications. If you know someone else that should be included here, please feel free to tag people.

Thank you for your precious time!

**Comment by poikilotherm** [comment]
Hey @timrobertson100 thanks for reaching out!
How about chatting about this lib on Matrix?

**Comment by timrobertson100** [comment]
Thanks @poikilotherm 

It's not something we're (GBIF) actively working on. I suspect we fixed up those two bugs we needed, sorted the stream handling and bumped some dependencies when we were in there. We'll be very happy to help get this pushed back to the master, although I suspect a committer will be able to cherry-pick the pieces very quickly anyway as we did so little.

We still have some dependency on J8 though, so perhaps a release using J8 to harmonise the forks, and then a J11 push would be sensible?
